### ISSUE TITLE
Add documentation about using Crick with Watson

### ISSUE THREAD

**Issue by jmaupetit** [TailorDev/Watson#165]
## Purpose

We recently open-sourced [Crick](https://github.com/TailorDev/crick) and there is only sparse documentation about how Watson & Crick talk to each other. Let's clarify this!

## Proposal

We should work on:

- [ ] Add basic documentation in the `README.md`
- [ ] Improve the `sync` command documentation
- [ ] Improve the backend section of `docs/user-guide/configuration.md`
- [ ] Improve the overview section of `docs/index.md`

**Comment by Hacktivista** [comment]
Yup! this is needed. I want to export watson data for creating my own view (not crick) and probably the sync command would help a lot, but I don't know how it works :(

Also, please try to not bloat the code of watson for coupling other tools, "high-cohesion, low-coupling". Thanks for this SW :)

**Comment by hacktivista** [comment]
Apparently crick is a dead project? Still we could use this functionality to extract data, plz let us know how

**Comment by jmaupetit** [comment]
Basically, the `sync` command performs a pull and then a push to a crick server. I invite you to read the code in the meantime:

- `sync`: https://github.com/TailorDev/Watson/blob/master/watson/cli.py#L1462
- `pull`: https://github.com/TailorDev/Watson/blob/master/watson/watson.py#L379
- `push`: https://github.com/TailorDev/Watson/blob/master/watson/watson.py#L409
